CHANGELOG.md

```diff
@@ -17,6 +17,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   code
 - README has been updated with example SSLContext setup and it's use in Conjur
   class constructors
+- Maven pom.xml file includes configuration profiles for deploying to Sonatype OSSRH 
+(OSS Repository Hosting)  ([cyberark/conjur-api-java#6](https://github.com/cyberark/conjur-api-java/issues/6))
```

Comment:
is this relevant to end users? if not it can be dropped

Reply:
Dropped